tests/phpunit/api/v3/ActivityTest.php

```diff
@@ -1218,28 +1206,27 @@ public function testActivityUpdateCheckCoreFields() {
     $params['assignee_contact_id'] = [$contact3 => $contact3];
     $params['target_contact_id'] = [$contact4 => $contact4];
 
-    $result = $this->callAPISuccess('activity', 'create', $params);
-
-    $this->assertEquals($activityId, $result['id'], ' in line ' . __LINE__);
```

Comment:
Also this one?

Reply:
Ok - in this case - the call to activity create was removed by accident

Reply:
Actually no - it was right to remove it - it's not doing an update - it's creating a random other activity that is not checked